javascript/implementation/Container.ts

```diff
@@ -125,11 +145,19 @@ export class Container extends Addressable {
             entryBuilder.setKey(wrapKey(key));
         } else if (Array.isArray(key)) {
             const pair = new PairBuilder();
-            pair.setLeft(muidToBuilder(key[0].address));
-            pair.setRite(muidToBuilder(key[1].address));
+            let key1 = key[0];
+            let key2 = key[1];
+            if ("address" in key[0] && "address" in key[1]) {
+                key1 = key[0].address;
+                key2 = key[1].address;
+            }
```

Comment:
addEntry taking only Addressables was very awkward since getEntryByKey takes Muids, but storageKeys are muidTuples. I found that the least insane change would just be to add the option to pass Muids to addEntry.